### PR TITLE
Coverage increase

### PR DESCRIPTION
**Pull request by floresj**
Updated unit tests

### CODE REVIEW

File: src/components/Form/Country/Country.test.jsx [hunk 1]
```diff
@@ -11,4 +11,23 @@ describe('The Country component', () => {
...
+      onFocus: () => {}
+    }
+    const component = mount(<Country {...expected} />)
+    component.find('input#country').simulate('change', {
+      target: {
+        value: 'Belg'
+      }
+    })
+    component.find('input#country').simulate('focus')
+    expect(component.find('.react-autosuggest__suggestions-list').length).toBeGreaterThan(0)
```
Comment (bmallred) [inline src/components/Form/Country/Country.test.jsx]: Nice

File: src/components/Form/RadioGroup/RadioGroup.test.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      </RadioGroup>
+    )
+
+    expect(component.find('input[type="radio"]').first().props().checked).toBe(true)
+  })
+
+  it('renders radio button children with selected func', () => {
+    let selectedValueFunc = (props) => {
+      return props.value === 'Option1'
+    }
```
Comment (bmallred) [inline src/components/Form/RadioGroup/RadioGroup.test.jsx]: I haven't had to use the `selectedValueFunc` yet but nice to have an example of usage

File: src/components/Section/Financial/Bankruptcy/Bankruptcy.test.jsx [hunk 3]
```diff
@@ -2,17 +2,70 @@ import React from 'react'
...
+  }
+]
+
 describe('The bankruptcy component', () => {
   it('no error on empty', () => {
     const expected = {
       name: 'bankruptcy'
     }
-    const component = mount(<Bankruptcy {...expected} />)
+    const component = mount(<Bankruptcy name={expected.name} />)
```
Comment (bmallred) [inline src/components/Section/Financial/Bankruptcy/Bankruptcy.test.jsx]: Is there a reason you went explicit on this? Just curious

File: src/components/Section/History/Employment/Supervisor.jsx [hunk 4]
```diff
@@ -136,6 +133,8 @@ export default class Supervisor extends ValidationElement {
             <Telephone name="Telephone"
               {...this.props.Telephone}
               label={i18n.t('history.employment.supervisor.telephone.label')}
+              onBlur={this.handleBlur}
+              onFocus={this.handleFocus}
```
Comment (bmallred) [inline src/components/Section/History/Employment/Supervisor.jsx]: I'm curious if we even need the `focus` and `blur` handlers here

**Comment by bmallred** [comment]
TravisCI reported it passed some time ago but it hasn't updated here. Going to go ahead with the merge.

## Results

travis-ci

File: src/components/Section/Financial/Bankruptcy/Bankruptcy.test.jsx (hunk 3 above)
Comment (floresj) [inline src/components/Section/Financial/Bankruptcy/Bankruptcy.test.jsx]: Ah I may have been doing something and never did an undo. No reason.

File: src/components/Section/History/Employment/Supervisor.jsx (hunk 4 above)
Comment (floresj) [inline src/components/Section/History/Employment/Supervisor.jsx]: This was a case to just cover the `handleFocus()`/`handleBlur()` funcs in there. But we could do away with those methods all together.

Comment (bmallred) [inline src/components/Section/History/Employment/Supervisor.jsx]: Yeah, I would imagine if they are not being actively used we shouldn't have them. We'll remove them from one of the working branches.

File: src/components/Section/Financial/Bankruptcy/Bankruptcy.test.jsx (hunk 3 above)
Comment (bmallred) [inline src/components/Section/Financial/Bankruptcy/Bankruptcy.test.jsx]: Cool, just curious if you preferred one or had some reasoning I wasn't aware of 😄

**Pull request merged**